pkg/workceptor/kubernetes.go

```diff
@@ -397,10 +404,16 @@ func (kw *kubeUnit) runWorkUsingLogger() {
 		streamWait.Done()
 	} else {
 		go func() {
-			errStdin = exec.Stream(remotecommand.StreamOptions{
-				Stdin: stdin,
-				Tty:   false,
-			})
+			for retries := 0; retries < 5; retries++ {
+				errStdin = exec.Stream(remotecommand.StreamOptions{
+					Stdin: stdin,
+					Tty:   false,
+				})
+				if errStdin != nil {
```

Comment:
Same question here - unless I am missing something won't we always run this 5 times?